mgrt/2.2-to-2.3.sql

```diff
@@ -0,0 +1,9 @@
+
+alter table group_memberships add column if not exists start_date timestamptz;
+alter table group_memberships add column if not exists end_date timestamptz;
+alter table group_memberships add column if not exists weekdays jsonb;
```

Comment:
Since `weekdays` is `jsonb`, it would be good if we spelled out the grammar used to specify constraints as part of the documentation. I noticed that there are examples in the tests further down, but ideally one wouldn't have to look at the code to write a constraint.

Reply:
More than the example at: https://github.com/unioslo/pg-iam/blob/6489df922a315fafa6df3e90e8a3ab89256bff6e/docs/3-example.md#set-up-group-memberships-and-moderators ?

Reply:
The examples are extensive. However I was thinking about a textual description/specification how the JSON ought to look like.

Reply:
Right, but where in the docs would you prefer?

Reply:
I think [here](https://github.com/unioslo/pg-iam/blob/6489df922a315fafa6df3e90e8a3ab89256bff6e/docs/2-db-structure.md?plain=1#L95) might be a good place. It's here that the signature of the method and the structure of the response are specified. All other arguments are, for the most part, self explanatory because of their scope or data type (`text`, `timestamptz`), whereas `json` is not.

It's here that I would look for advice as to what goes into and comes out of `group_member_add()`.

Reply:
https://github.com/unioslo/pg-iam/pull/21/commits/b8165caed2b5cd3d4557c761a0bcae99a02ab6b7